modules/storage-persistent-storage-pv.adoc

```diff
@@ -66,15 +66,15 @@ endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 == Capacity
 
 Generally, a PV has a specific storage capacity. This is set by using the
-PV's `capacity` attribute.
+`PersistentVolume` object's `capacity` attribute.
```

Comment:
s/`PersistentVolume` object's `capacity` attribute./`capacity` attribute of the PV.